app/models/reporting/return.rb

```diff
@@ -125,7 +127,7 @@ def stays
         transaction do
           self.class.where(client_id: ids).delete_all
           self.class.import(headers, batch_of_stays.map(&:values))
-          @notifier.ping("Return: Adding #{batch_of_stays.count} returns")
+          @notifier.ping("Return: Adding #{batch_of_stays.count} returns") if send_ping
```

Comment:
I feel like this should tell us how many returns were added since the last report, not just the size of that batch...
Right now it will say something like
```
Return: Starting batch 1 in batches of 1,000 of 30000 total clients
Return: Adding 700 returns
```
Ignoring all the returns that were in the previous 24 batches.
Maybe something like
```suggestion
          if send_ping
            @notifier.ping("Return: Added #{added} returns")
            added = 0
          else
            added += batch_of_stays.count
          end
```